### ISSUE TITLE
[Bug]: unexpected css transform for color

### ISSUE THREAD

**Issue by fz6m** [web-infra-dev/rspack#7747]
### System Info

```bash
System:
    OS: macOS 14.2.1
    CPU: (8) arm64 Apple M3
    Memory: 114.19 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.19.0 - ~/Library/Caches/fnm_multishells/26058_1725135681510/bin/node
    Yarn: 1.22.22 - ~/Library/Caches/fnm_multishells/26058_1725135681510/bin/yarn
    npm: 10.2.3 - ~/Library/Caches/fnm_multishells/26058_1725135681510/bin/npm
    pnpm: 9.7.1 - ~/Library/Caches/fnm_multishells/26058_1725135681510/bin/pnpm
    bun: 1.1.5 - ~/.bun/bin/bun
    Watchman: 2024.07.08.00 - /opt/homebrew/bin/watchman
  Browsers:
    Chrome: 128.0.6613.114
    Safari: 17.2.1
```

### Details

rspack and webpack handle css differently. 

see [minimal reproduction](https://github.com/MoeYc/rspack-unexpected-css-transform).

### Reproduce Steps

See readme

**Comment by jerrykingxyz** [comment]
It seems be transformed by lightningcss when minimize css. You can try it in [playground](https://lightningcss.dev/playground/index.html#%7B%22minify%22%3Atrue%2C%22customMedia%22%3Atrue%2C%22cssModules%22%3Afalse%2C%22analyzeDependencies%22%3Afalse%2C%22targets%22%3A%7B%22chrome%22%3A6225920%7D%2C%22include%22%3A0%2C%22exclude%22%3A0%2C%22source%22%3A%22.a%20%7B%5Cn%20%20background%3A%20rgba(0%2C0%2C0%2C0.5)%3B%5Cn%7D%22%2C%22visitorEnabled%22%3Afalse%2C%22visitor%22%3A%22%7B%5Cn%20%20Color(color)%20%7B%5Cn%20%20%20%20if%20(color.type%20%3D%3D%3D%20'rgb')%20%7B%5Cn%20%20%20%20%20%20color.g%20%3D%200%3B%5Cn%20%20%20%20%20%20return%20color%3B%5Cn%20%20%20%20%7D%5Cn%20%20%7D%5Cn%7D%22%2C%22unusedSymbols%22%3A%5B%5D%2C%22version%22%3A%22local%22%7D)

If you want to avoid this behavior, you can set the target in LightningCssMinimizerRspackPlugin.
https://rspack.dev/plugins/rspack/lightning-css-minimizer-rspack-plugin#minimizeroptions

**Comment by fz6m** [comment]
fixed in #7811
